### PR TITLE
update readme and issue template 

### LINKED ISSUE #97: Meta Package explanation on readme?
My guess is that people thinks this is the repo one should go to for binder - based on my observations from our similar meta-package https://github.com/jupyter/jupyter.  Should we increase the explanation of this package in the README and link to binderhub?

### PR DESCRIPTION
**Pull request by jzf2101**
to explain to users the purpose of the repo- closes #97

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,4 +2,18 @@
...
+
+**For the main binder repository, visit [binderhub](https://github.com/jupyterhub/binderhub)**
+
+This repository is the metapackage for [binder](https://mybinder.org) usage, docs, and chat.
+
+ Major binder development happens on [binderhub](https://github.com/jupyterhub/binderhub).  The majority of issues related to binder at this repo.  Please refer to BinderHub for further details on the binder project.
+
+## Additional repos
+
+ For deployment of the website [mybinder.org](https://mybinder.org), visit [mybinder.org-deploy](https://github.com/jupyterhub/mybinder.org-deploy).  Helm charts are at [helm-chart](https://github.com/jupyterhub/helm-chart).
```
Comment (choldgraf) [inline README.md]: `BinderHub` uses technology called "Helm" to configure and manage deployment of the Binder service. For details about this deployment, see [helm-chart]...` etc

File: README.md [hunk 2]
```diff
@@ -2,4 +2,18 @@
...
+
+This repository is the metapackage for [binder](https://mybinder.org) usage, docs, and chat.
+
+ Major binder development happens on [binderhub](https://github.com/jupyterhub/binderhub).  The majority of issues related to binder at this repo.  Please refer to BinderHub for further details on the binder project.
+
+## Additional repos
+
+ For deployment of the website [mybinder.org](https://mybinder.org), visit [mybinder.org-deploy](https://github.com/jupyterhub/mybinder.org-deploy).  Helm charts are at [helm-chart](https://github.com/jupyterhub/helm-chart).  
+
+ Binder is maintained by the [JupyterHub team](https://github.com/jupyterhub/team-compass#jupyterhub-team).  Administration is managed at [team compass](https://github.com/jupyterhub/team-compass).
```
Comment (choldgraf) [inline README.md]: seems like it'd make sense to point this link to the binder team link (https://github.com/jupyterhub/team-compass#binder-team) no?

Comment (choldgraf) [reply README.md]: we can mention and link to the jupyterhub team too though

File: README.md [hunk 3]
```diff
@@ -2,4 +2,18 @@
...
+
+
+The JupyterHub team also maintains analysis of binder data including [billing data](https://github.com/jupyterhub/binder-billing) and [activity data](https://github.com/jupyterhub/binder-data).
```
Comment (choldgraf) [inline README.md]: IMO we could link to binder-billing, but I don't know that binder-data is in a state where we'd want to advertise. Do you think it is?

File: .github/ISSUE_TEMPLATE.md [hunk 4]
```diff
@@ -1,13 +1,13 @@
...
-* ... https://mybinder.org site downtime or inaccessible pages: https://github.com/jupyterhub/mybinder.org-deploy/issues/new
-* ... the software powering all this (binderhub) in general: https://github.com/jupyterhub/binderhub/issues/new
+Help us stay on top of issues by submitting them to the right repository.  This repo is best for issues related to:
+* repository build issues
+* general questions
 
-If you aren't sure where to go use https://github.com/jupyterhub/binder/issues/new.
+For issues related to:
+* ... mybinder.org site downtime or inaccessible pages visit: https://github.com/jupyterhub/mybinder.org-deploy/issues/new
+* ... binder software visit: https://github.com/jupyterhub/binderhub/issues/new
```
[the remark this answers is not in the document]
Comment (willingc) [reply .github/ISSUE_TEMPLATE.md]: ~~visit~~

Perhaps: "BinderHub for issues related to hosting your own Binder service:"

File: README.md [hunk 5]
```diff
@@ -2,4 +2,18 @@
```
Comment (willingc) [inline README.md]: Add:

"This repository contains the documentation and usage instructions for the mybinder.org service"

File: README.md [hunk 6]
```diff
@@ -2,4 +2,18 @@
 
 [![Join the chat at https://gitter.im/jupyterhub/binder](https://badges.gitter.im/jupyterhub/binder.svg)](https://gitter.im/jupyterhub/binder?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
```
Comment (willingc) [inline README.md]: ## Communication

Join the chat...

Comment (willingc) [reply README.md]: Before Line 3

File: README.md [hunk 7]
```diff
@@ -2,4 +2,18 @@
...
 
-Binder metapackage for usage, docs, and chat
+
+**For the main binder repository, visit [binderhub](https://github.com/jupyterhub/binderhub)**
+
+This repository is the metapackage for [binder](https://mybinder.org) usage, docs, and chat.
```
Comment (willingc) [inline README.md]: Move this above Line 6:

## Repositories

Rest of text...

File: README.md [hunk 8]
```diff
@@ -2,4 +2,18 @@
...
+
+ Major binder development happens on [binderhub](https://github.com/jupyterhub/binderhub).  The majority of issues related to binder at this repo.  Please refer to BinderHub for further details on the binder project.
```
[the remark this answers is not in the document]
Comment (willingc) [reply README.md]: I would strike this line and replace with:

For information about the `mybinder.org` service or to report an issue with it, see ...

File: README.md (hunk 3 above)
Comment (jzf2101) [reply README.md]: It will be eventually IMHO

**Comment by choldgraf** [comment]
woo thanks! 🎉

**Pull request merged**